common/filetoolkitwithundo.cpp

```diff
@@ -127,7 +127,7 @@ bool FileToolkitWithUndo::bindMount(const std::string &src,
 
     log_debug() << "Bind-mounting " << src << " in " << dst << ", flags: " << flags;
 
-    if(enableWriteBuffer) {
+    if(enableWriteBuffer && isDirectory(src)) {
```

Comment:
So this means that the bindmounted files will not use overlayfs at all? If so then this should be mentioned in the documentation I guess.